src/APIInterfaces/BackendAPI/Requests/CourseRequests.ts

```diff
@@ -722,6 +722,17 @@ export const deleteAttachments = async ({
     }
 };
 
+export const startExportOfTopic = async ({
+    topicId,
+    force = false,
+}: {topicId: number; force: boolean}): Promise<AxiosResponse<any>> => {
+    try {
+        return await AxiosRequest.post(url.resolve(COURSE_TOPIC_PATH, `${topicId}/startExport?force=${force}`));
```

Comment:
Should use axios configuration for params or query stringify, not doing it by hand